js/main.js

```diff
@@ -0,0 +1,70 @@
+
+
+var myButton = document.querySelector("#donateBike");
+
+myButton.addEventListener("click", doSomething);
+
+function doSomething() {
```

Comment:
try to use a name that defines what the function does. Same goes for doSomething2